iFixit/react-commerce
# Empty facet state:  Add grayed out facets

## Summary

Previously, we had a weird empty facet state when we searched for a product where having any facets didn't make sense. 
This PR adds grayed out facets when we end up in an empty facet situation.

Before:
![Screenshot 2022-07-20 134723](https://user-images.githubusercontent.com/50181909/180083552-a41df47d-8b0d-46b0-b8a5-67dbb88ce3cb.png)


Now:
![Screenshot 2022-07-20 141756](https://user-images.githubusercontent.com/50181909/180083905-e23ed42b-a199-4b05-8854-d0f60f65151d.png)

## QA:
- Visit [the preview](https://react-commerce-git-empty-facet-state-add-grayed-o-f37252-ifixit.vercel.app/Tools) 
- I was able to find an empty facet product under `/Tools` but there can be others. This is an example of one:
   - In the search box, look up `halberd spudger` [same as the screenshots above]
   - The grayed out facets should now appear
   - I also looked up something that's irrelevant to `/Tools` [`battery iphone` for example] to make sure that behavior hasn't changed and is the same as before.

Closes #461 

CC: @sterlinghirsh
